semgrep/semgrep/__init__.py

```diff
@@ -1 +1 @@
-__VERSION__ = "0.20.0"
+__VERSION__ = "0.20.0-dev"
```

Comment:
Did you mean to update this and pfff?

Reply:
No I did not, in fact I meticulously worked to ensure that didn't happen and it seemed to have happened anyway :(